src/text.rs

```diff
@@ -195,53 +232,104 @@ impl<'a> From<&'a str> for Span<'a> {
 
 /// A string composed of clusters of graphemes, each with their own style.
 #[derive(Debug, Clone, PartialEq, Default, Eq)]
-pub struct Spans<'a>(pub Vec<Span<'a>>);
+pub struct Line<'a>(pub Vec<Span<'a>>);
 
```

Comment:
Do you think it's worth leaving a type alias around so that consumers are not broken. E.g.:
```suggestion

#[deprecated(since = "0.21.0", note = "`Spans` is renamed to `Lines` and will be removed in some future release"]
pub type Spans = Lines;

```

Reply:
No, I think we should have them use `Line` both for consistency in examples/codebases and also readability of their own code. It's not a particularly difficult refactor. Though I understand where you're coming from.

Reply:
This is for current consumers and future migration, not for internal use.
By breaking this here, you make it impossible for a consumer of this library to migrate from tui or to update from 0.20.1 to 0.21.0. There are around 3500 files from a quick github search where that causes a problem. It's not difficult to make each change, but the impact is large. Why foist all that work onto others when it can be avoided or deferred? By adding the type alias, you have the best of both worlds - communication that something will break, instruction on how to fix it, backwards compatible change,  and it's all delivered via the compiler.

Reply:
I didn't read this carefully. I think that marking it as deprecated is a good solution.